sql/logon.sql

```diff
@@ -28,8 +28,8 @@ CREATE TABLE IF NOT EXISTS `account` (
   `failed_logins` int(11) unsigned NOT NULL DEFAULT '0',
   `locked` tinyint(3) unsigned NOT NULL DEFAULT '0',
   `lock_country` varchar(2) NOT NULL DEFAULT '00',
-  `last_login` timestamp NOT NULL DEFAULT '0000-00-00 00:00:00',
-  `last_pwd_reset` timestamp NOT NULL DEFAULT '0000-00-00 00:00:00',
+  `last_login` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `last_pwd_reset` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,

```

Comment:
SQL Error (1293): Incorrect table definition; there can be only one TIMESTAMP column with CURRENT_TIMESTAMP in DEFAULT or ON UPDATE clause

Reply:
let check on this, this "zero date" cause an error while applying the migration so i attempted to change it.